Backend/app/dashboard/rider-dashboard/src/API/RiderPlatform.hs

```diff
@@ -56,6 +57,7 @@ type API' =
     :<|> Tickets.API
     :<|> HotSpot.API
     :<|> ManagementAPI
+    :<|> RideBookingAPI
```

Comment:
It should be exposed from Management APIs, not directly.

Reply:
Ohh yes, this was wrongly added in RideBooking first so forgot to remove from here, Removing.